cpp/src/parquet/statistics_test.cc

```diff
@@ -775,6 +785,10 @@ class TestStatisticsHasFlag : public TestStatistics<TestType> {
     EXPECT_FALSE(encoded.has_distinct_count);
     EXPECT_FALSE(encoded.has_min);
     EXPECT_FALSE(encoded.has_max);
+    EXPECT_FALSE(encoded.is_min_value_exact.has_value());
+    EXPECT_FALSE(encoded.is_max_value_exact.has_value());
+    EXPECT_EQ(std::nullopt, encoded.is_min_value_exact);
+    EXPECT_EQ(std::nullopt, encoded.is_max_value_exact);
```

Comment:
You don't need *both* tests. `has_value` returns false iff the optional is equal to `std::nullopt`.